### PR TITLE
navbar and footer added, no text added yet

### LINKED ISSUE #4: Implement Figma Prototype into Frontend

### PR DESCRIPTION
**Pull request by austinwli**

### CODE REVIEW

File: react-frontend-app/src/components/MainNavBar.js [hunk 1]
```diff
@@ -1,23 +1,27 @@
...
-    }
-
-    render () {
-        return (
-            <div>
-                <p>I am the Zest Navigation Bar (purely cosmetic until we have multiple pages). I'm very sad b/c I have no code here yet</p>
-            </div>
-        )
-    }
+function MainNavBar() {
```
Comment (dzheng21) [inline react-frontend-app/src/components/MainNavBar.js:4]: You really love functional React hahahah

File: react-frontend-app/src/App.js [hunk 2]
```diff
@@ -11,7 +11,6 @@ import logo from './logo.svg';
 import './style/App.css';
 
 // Import different components
-import Description from './components/Description'
```
Comment (dzheng21) [inline react-frontend-app/src/App.js]: Any reason why this is booted? I think it's chill to not have Description as a separate component, but just to be on the same page.

File: react-frontend-app/src/components/MainNavBar.js (hunk 1 above)
Comment (austinwli) [reply react-frontend-app/src/components/MainNavBar.js:4]: ;)

File: react-frontend-app/src/App.js (hunk 2 above)
Comment (austinwli) [reply react-frontend-app/src/App.js]: oh yeah i meant to delete all the description files since we could just paste the fat paragraph into the app.js

Comment (dzheng21) [reply react-frontend-app/src/App.js]: Sounds good, unsure if we will need it again down the road but that works for me.  Ideally there would be an easy place for Zest to access and edit, if we can make it as clear as possible that would be optimal

**Review by dzheng21 — APPROVED** [review]
Looks great! There may be some merge conflicts w/ you and Kayla's PR, but just to note.

**Review by chrisong88 — APPROVED** [review]
Great work overall on a big array of challenges! 

Small point for down the line: since you have a media query at 864px, are you planning on adding additional media queries for mobile users at around 480px and such?

**Review by kaylanhua — APPROVED** [review]

**Pull request merged**